### PR TITLE
Russell-L2111.ag_nonco2_IRR

### PR DESCRIPTION
**Pull request by russellhz**
#552 

Currently have L2111.AnEmissions and L2111.AnNH3Emissions removed from this

### CODE REVIEW

**Comment by bpbond** [comment]
@cahartin Please review or assign a reviewer. Thanks.

File: R/zchunk_L2111.ag_nonco2_IRR.R [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+        add_title("Agricultural Emissions") %>%
+        add_units("Tg") %>%
+        add_comments("Added region to AGR emissions") %>%
+        add_legacy_name("L2111.AGREmissions") %>%
+        add_precursors("temp-data-inject/L1221.ghg_tg_R_agr_C_Y_GLU_IRR", "water/basin_to_country_mapping",
+                       "common/GCAM_region_names") ->
+        L2111.AGREmissions
+      L2111.AGRBio %>%
+        add_title("Bio N2O Coefficients by region and technology") %>%
+        add_units("Unitless") %>% # should this be labeled in emissions/A_regions
```
Comment (cahartin) [inline R/zchunk_L2111.ag_nonco2_IRR.R]: @russellhz what do you mean by this comment?

Comment (russellhz) [inline R/zchunk_L2111.ag_nonco2_IRR.R]: The data originated from A_regions, but the units aren't labeled there, so I wasn't sure what to put. I opened an issue (#559). I'm not sure I actually meant to leave that comment in though.

Comment (cahartin) [inline R/zchunk_L2111.ag_nonco2_IRR.R]: I say delete the comment wait for a resolution on the issue of the units.

**Comment by russellhz** [comment]
Switched to some module-helpers functions and changed unit in response to #559

**Comment by bpbond** [comment]
Holding on @cahartin approval.

**Pull request merged**